nixos/roles/lamp.nix

```diff
@@ -0,0 +1,157 @@
+{ config, pkgs, lib, ... }:
+let
+  role = config.flyingcircus.roles.lamp;
+
+in {
+  options = {
+    flyingcircus.roles.lamp.enable =
+      lib.mkEnableOption "Flying Circus LAMP stack";
+  };
+
+  config = lib.mkIf role.enable (
+    let
+      apacheConfFile = /etc/local/lamp/apache.conf;
+      phpiniConfFile = /etc/local/lamp/php.ini;
+
+      services.httpd.user = "s-vmonitor";
+
+      phpOptions = ''
+          extension=${pkgs.php73Packages.memcached}/lib/php/extensions/memcached.so
+          extension=${pkgs.php73Packages.imagick}/lib/php/extensions/imagick.so
+          extension=${pkgs.php73Packages.redis}/lib/php/extensions/redis.so
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; General settings
+
+          output_buffering = On
+          short_open_tag = On
+          curl.cainfo = /etc/ssl/certs/ca-certificates.crt
+          sendmail_path = /var/setuid-wrappers/sendmail -t -i
```

Comment:
This only works on 15.09. It's /run/wrappers/bin now.

Reply:
merci. also that reminded me off adding the mailstub to the RGs ...